src/components/Editor/AvatarParticipationStatus.vue

```diff
@@ -145,18 +146,11 @@ export default {
 				}
 			}
 
-			if (this.isViewedByOrganizer) {
-				return {
-					indicatorClass: ['no-response', 'icon', 'icon-invitees-no-response-white'],
-					label: t('calendar', 'Invitation sent'),
-				}
-			} else {
-				return {
-					indicatorClass: ['no-response', 'icon', 'icon-invitees-no-response-white'],
-					label: t('calendar', 'Has not responded to {organizerName}\'s invitation yet', {
-						organizerName: this.organizerDisplayName,
-					}),
-				}
+			return {
```

Comment:
Could you briefly explain why the check for the isViewedByOrganiser isn't needed any more?

Reply:
> Could you briefly explain why the check for the isViewedByOrganiser isn't needed any more?

Without this fix, when you create an appointment, calendar will display "Invitation sent" just after attendee is added, before any messages are sent/event saved. This is misleading.

Reply:
Yes that is true, but the visibility of the email sending status for the organizer **only** should still be kept in.

Reply:
Nextcloud should not lie to user (now it lies that invitation was sent even before event is saved and before sending may take place; more - it also lies about sent invitation if sending invitations is disabled in Nextcloud...). Nextcloud cannot state that invitation was not sent (i.e. when sending invitations is disabled in Nextcloud) because external client (like Thunderbird) could have done it already. Solution proposed here sounds most reasonably for us. Changing it requires more work (i.e. update status after message was actually sent by Nextcloud) and is outside of scope of this PR.

Reply:
That is not what I'm talking about at all, I agree with you on having a correct state on the email sending. I am talking about the VISIBILITY of the element. It should only be visible to the Organizer of the event. I cannot find this limitation in your code as you have removed it. Does your new code cover this visibility issue in a different way or does this revert previous work?

Reply:
> Does your new code cover this visibility issue in a different way or does this revert previous work?

After this PR organizes sees same message as any other user ("Has not responded to {organizerName}\'s invitation yet").

Reply:
ok then this code change will remove the limitation. I would ask you to please take a look at the old code and add this back in in a way that works with your changes. @st3iny can you also please have a look at this?

Reply:
This PR displays "Has not responded to..." for ALL users so no need for PR to be updated (code from else was moved outside of removed if/else).